loopchain/baseservice/broadcast_scheduler.py

```diff
@@ -390,12 +391,43 @@ def schedule_job(self, command, params, block=False, block_timeout=None):
         self._put_command(command, params, block=block, block_timeout=block_timeout)
         self.__perform_schedule_listener(command, params)
 
-    def schedule_broadcast(self, method_name, method_param, *, retry_times=None, timeout=None):
+    def _update_audience(self, reps_hash, update_command=None):
+        blockchain = ObjectManager().channel_service.block_manager.blockchain
+
+        if update_command:
+            update_reps = blockchain.find_preps_by_roothash(reps_hash)
+            util.logger.info(
+                f"update audience command({update_command})"
+                f"\nupdate_reps({update_reps})"
+            )
+            for rep in update_reps:
+                self.schedule_job(update_command, rep['p2pEndpoint'])
```

Comment:
scheduling `BroadcastCommand.SUBSCRIBE` and `BroadcastCommand.UNSUBSCRIBE` can be changed to methods like add_audience / remove_audience in `broadcast_scheduler`, because it doesn't actually broadcast anymore.

Reply:
These names originate from the old p2p framework, but your point of view is very reasonable. We will consider your proposal in the another PR.